packages/common/test/content/edit.test.ts

```diff
@@ -227,6 +227,13 @@ describe("content editing:", () => {
       expect(updateModelSpy.calls.count()).toBe(1);
       expect(getServiceSpy).toHaveBeenCalledTimes(1);
       expect(updateServiceSpy).toHaveBeenCalledTimes(1);
+      expect(updateServiceSpy).toHaveBeenCalledWith(
+        "https://services.arcgis.com/:orgId/arcgis/rest/services/:serviceName/FeatureServer",
+        {
+          authentication: MOCK_AUTH,
+          updateDefinition: { capabilities: "Query,Extract" },
+        }
+      );
       const [url, { updateDefinition }] = updateServiceSpy.calls.argsFor(0);
```

Comment:
Heads up that this check is now redundant with the one you just introduced. You may also want to change this test case so the `getService` spy returns an additional field that `updateService` won't see.

Reply:
Good call and suggestion! I have made the change you suggested.